src/test/java/com/wikia/webdriver/common/remote/operations/ValidatePost.java

```diff
@@ -0,0 +1,20 @@
+package com.wikia.webdriver.common.remote.operations;
+
+import com.google.common.collect.ImmutableMap;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.remote.Discussions;
+import com.wikia.webdriver.common.remote.context.ModeratePostContext;
+import org.json.JSONObject;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.json.JSONObject'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)